game/webserver.cc

```diff
@@ -96,7 +99,23 @@ http_server::response WebServer::GETresponse(const http_server::request &request
 		return http_server::response::stock_reply(http_server::response::ok, jsonRoot.toStyledString());
 	} else if(request.destination == "/api/getplaylistTimeout") {
 		return http_server::response::stock_reply(http_server::response::ok, std::to_string(config["game/playlist_screen_timeout"].i()));
-	} else {
+	} else if(request.destination.find("/api/language") == 0) {
+		map<std::string, std::string> localeMap = GenerateLocaleDict();
+		
+		Json::Value jsonRoot = Json::objectValue;
+		for (auto const &kv : localeMap) {
+			std::string key = kv.first;
+			//Hack to get an easy key value pair within the json object.
+			if(key == "Web interface by Niek Nooijens and Arjan Speiard, for full credits regarding Performous see /docs/Authors.txt"){
```

Comment:
@nieknooijens One problem here, docs/authors.txt does not live within the scope of the webserver, therefore is not reachable or we should symlink it to the www folder, i'm not quite fond of that..